responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/security-phrase/security-phrase.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { CustomValidationService } from 'src/app/core/services/customValidation.service';
+import { StepCreateEssFileService } from '../../step-create-ess-file/step-create-ess-file.service';
+
+import * as globalConst from '../../../../core/services/global-constants';
+import { MatCheckboxChange } from '@angular/material/checkbox';
+import { FormBuilder, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-security-phrase',
+  templateUrl: './security-phrase.component.html',
+  styleUrls: ['./security-phrase.component.scss']
+})
+export class SecurityPhraseComponent implements OnInit, OnDestroy {
+  securityPhraseFC: FormControl = null;
+  bypassPhrase: FormControl = null;
+  tabUpdateSubscription: Subscription;
+
+  constructor(
+    private stepCreateEssFileService: StepCreateEssFileService,
+    private customValidationService: CustomValidationService,
+    private router: Router
+  ) {}
+
+  ngOnInit(): void {
+    // Set up form validation for verification check
+    this.securityPhraseFC = new FormControl(
```

Comment:
Use form group to enclose both securityPhraseFC and bypassPhrase controls

Reply:
BypassPhrase has to be separated from SecurityPhrase, the two are mutually exclusive. If they're grouped, we'd have to add a bunch of extra code to make them not act like a group.

Reply:
Please add conditional validation to control the mutually exclusive behavior. All user input fields needs to be part of a form. I just checked profile/SecurityQuestionsComponent, :
this.bypassQuestions = new FormControl(
      this.stepCreateProfileService.bypassSecurityQuestions
    );
this needs to be moved to formgroup as well. My bad, I missed it during the review of this component.

Reply:
I don't like this, but for solidarity with the rest of the code, I'll group the loose FormControls on Security Phrase, Security Question, and Review & Save.